internal/pkg/service/buffer/api/service/import.go

```diff
@@ -1,76 +1,15 @@
 package service
 
 import (
-	"fmt"
 	"io"
-	"net/http"
 
 	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/api/dependencies"
 	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/api/gen/buffer"
-	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/api/receive"
-	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/api/receive/receivectx"
 	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/store/key"
-	. "github.com/keboola/keboola-as-code/internal/pkg/service/common/errors"
-	"github.com/keboola/keboola-as-code/internal/pkg/utils/errors"
 )
 
 func (s *service) Import(d dependencies.ForPublicRequest, payload *buffer.ImportPayload, bodyReader io.ReadCloser) (err error) {
-	ctx, str := d.RequestCtx(), d.Store()
-
```

Comment:
V `service` zostala posledna logika, pre import endpoint.

Presunul som ju do `receive.Importer` (kde je uz zbytok ohladom import endpoint).

Hlavny dovod je, ze cele konfiguracne API pouziva `store.Store`, ale iba import endpoint `watche.APINode`. No v `service` boli dostupne obe, tak aby to niekto omylom nepouzil nespravne.